packages/disputer/test/Disputer.js

```diff
@@ -28,16 +29,6 @@ const {
   DSProxyManager,
 } = require("@uma/financial-templates-lib");
 
-// Run the tests against 3 different kinds of token/synth decimal combinations:
-// 1) matching 18 & 18 for collateral for most token types with normal tokens.
-// 2) non-matching 8 collateral & 18 synthetic for legacy UMA synthetics.
-// 3) matching 8 collateral & 8 synthetic for current UMA synthetics.
-const configs = [
```

Comment:
no point in re-defining this set of configs. this was pulled to common util and refactored over all files that did multi-decimal tests